### PR TITLE
Upgrade to use the latest Tika version

### PR DESCRIPTION
**Pull request by SS1L**

### CODE REVIEW

**Review by vladgolubev — APPROVED** [review]

**Comment by vladgolubev** [comment]
Please update readme stating that 3.x version works with Tika 2.2 only 

And users should use 2.x package version for tika 1.x

File: basic-usage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const testFile = require('fs').readFileSync('./Readme.md');
+const tte = require('./lib').default;
+const options = {
+  alignWithJava8: true,
+  executableJavaPath: 'java/lib/jvm/jre-1.8.0-openjdk-1.8.0.265.b01-1.lambda2.0.1.x86_64/bin/java',
+  firstVersionOfTika: true,
```
Comment (randomhash) [inline basic-usage.js:6]:
```suggestion
  useTikaV1: true,
```

File: src/server.ts [hunk 2]
```diff
@@ -53,3 +52,13 @@ function getOptionsBasedOnJavaVersion(options: TextExtractionConfig): string {
 
   return '--add-modules=java.xml.bind,java.activation';
 }
+
+function checkTikaVersion(artifactPath: string, options?: TextExtractionConfig): string {
+  if (options?.firstVersionOfTika) {
```
Comment (randomhash) [inline src/server.ts:57]:
```suggestion
  if (options?.useTikaV1) {
```

**Comment by randomhash** [comment]
@SS1L please update readme with a guidence how to use it with v1 further and that v3 default expects v2 tika

File: basic-usage.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const testFile = require('fs').readFileSync('./Readme.md');
```
Comment (randomhash) [inline basic-usage.js:1]: please move this file into "examples" folder

**Pull request merged**